src/components/buttons/pages/buttons-interactive.jsx

```diff
@@ -93,7 +93,7 @@ const Buttons = () => {
   return (
     <div>
       <DocsActiveBlock settings={buttonsSettings}>
-        <ButtonIconAligner type={BUTTON_TYPE.SOLID_BLUE}>
+        <ButtonIconAligner type={BUTTON_TYPE.SOLID}>
```

Comment:
@coderitual @domidomi @clxandstuff 
This wrapper is displayed as a code example in our docs:
<img width="706" alt="Screenshot 2022-04-26 at 13 25 24" src="https://user-images.githubusercontent.com/60467496/165289942-a32fcea3-9d2c-4b15-9a67-d846fc6f7501.png">
It would be great to have a real Button example here, but since we're going to deprecate this doc, maybe we should just ignore it?

Reply:
I think it can stay as it is. I tried to be minimal when it comes to our current docs, but of course I don't want to make user experience worse, so I can change it.